src/features/todos.ts

```diff
@@ -1,8 +1,12 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    setTodos(_, { payload }: PayloadAction<Todo[]>) {
+      return payload;
+    },
```

Comment:
You're completely replacing the state with payload in the `setTodos` reducer. This is not recommended in Redux. Instead, you should make changes to the existing state. For instance, you can add new todos to the existing state.